src/ecdsa/ecdh.py

```diff
@@ -169,47 +211,58 @@ def load_received_public_key(self, public_key):
 
     def load_received_public_key_bytes(self, public_key_str):
         """
-        Load public key from bytes string.
+        Load public key from byte string.
         Uses current curve and checks if key length corresponds to
         the current curve.
+
         :param public_key_str: public key in bytes string format
+        :type public_key_str: byte string
         :return: none
         """
         return self.load_received_public_key(
             VerifyingKey.from_string(public_key_str, self.curve))
 
     def load_received_public_key_der(self, public_key_str):
         """
-        Load public key from DER binary string.
-        Curve is get from DER and check if it the same as current curve
+        Load public key from DER byte string.
+        Compares the curve of the DER-encoded key with the ECDH set curve,
+        uses the former if unset.
 
         Note, the only DER format supported is the RFC5912
         Look at keys.py:VerifyingKey.from_der()
 
-        :param public_key_str: binary string with the DER encoding of public ECDSA key
+        :param public_key_str: byte string with the DER encoding of public ECDSA key
+        :type public_key_str: byte string
```

Comment:
`public_key_str` names of parameters are part of public API, so I think dropping the `_str` part would make it cleaner